tensorboard/components/vz_example_viewer/vz-example-viewer.ts

```diff
@@ -1071,7 +1074,7 @@ Polymer({
    * be used in css classes/ids.
    */
   sanitizeFeature: function(feat: string) {
-   let sanitized = feat;
+   let sanitized = feat.trim();
```

Comment:
nit: indentation looks off

Reply:
Just saying, because of L1079, `sanitized` may still have the whitespace. Perhaps use `sanitized` in L1078 and L1079?

Reply:
you're right, thx